applications/StructuralMechanicsApplication/custom_elements/membrane_element.hpp

```diff
@@ -0,0 +1,415 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Klaus B. Sautter
+//
+
+#if !defined(MEMBRANE_ELEMENT_3D_H_INCLUDED )
+#define  MEMBRANE_ELEMENT_3D_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/element.h"
+#include "custom_utilities/structural_mechanics_math_utilities.hpp"
+
+namespace Kratos
+{
+
+  class MembraneElement
+    : public Element
+  {
+  public:
+
+    // Counted pointer of MembraneElement
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(MembraneElement);
+
+    // Constructor using an array of nodes
+    MembraneElement(IndexType NewId, GeometryType::Pointer pGeometry);
+
+    // Constructor using an array of nodes with properties
+    MembraneElement(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties);
+
+    // Destructor
+    ~MembraneElement() override;
```

Comment:
```suggestion
    ~MembraneElement() override = default;
```

imo not needed to pollute the cpp with an empty destructor